Driver/Driver/Scenes/SplashScreen/SplashScreenViewModel.swift

```diff
@@ -14,13 +14,30 @@ final class SplashScreenViewModel: SplashScreenViewModelProtocol {
     var router: SplashScreenRouterProtocol!
   
     // MARK: Functions
-    func getSomeData() {
-        DispatchQueue.main.asyncAfter(deadline: .now() + 4) { [weak self] in
-            guard let self else { return }
-            self.delegate?.handleOutput(.returnData(someData: "Done"))
+    func getToken() {
+        
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
```

Comment:
2 seconds are so long. why did you use this?

Reply:
I forgot to change it thanks for your caution